cscs-checks/microbenchmarks/cpu/roofline/berkeley-ert.py

```diff
@@ -0,0 +1,394 @@
+# Copyright 2016-2021 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import os
+import sys
+import reframe as rfm
+import reframe.utility.sanity as sn
+import reframe.utility.udeps as udeps
+
+# {{{ readme:
```

Comment:
@jgphpc, is there any reason why you have this `# {{{` and `# }}}` syntax on the file?

Reply:
yes, it folds the lines in between, when using vim.
for non vim users, it's just a comment line to be ignored.
It improves readability so I would prefer to keep it.